src/Storages/MergeTree/Compaction/PartsFinders/IPartsFinder.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <Storages/MergeTree/FutureMergedMutatedPart.h>
+
+#include <Storages/MergeTree/Compaction/MergeSelectorApplier.h>
+
+namespace DB
+{
+
+class IPartsFinder
```

Comment:
Maybe we don't need this interface at all?

Reply:
removed finder completely